Extensions/Effects/JsExtension.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
           'Adjust gamma, contrast, saturation, brightness, alpha or color-channel shift.'
         )
       )
-      .addIncludeFile('Extensions/Effects/pixi-filters/filter-adjustment.js')
+      .addIncludeFile('Extensions/Effects/pixi-filters/pixi-filters.js')
```

Comment:
Could we keep the filters in separate files? Otherwise this means that a game using one of these filters will pay the cost of including **all** other filters. Meaning a larger file to download, a slightly increased memory usage/parsing cost for the JS engine.

Having separate files for filters allow you to just "pay for what you use" :)